de.dlr.sc.virsat.model.extension.statemachines/src/de/dlr/sc/virsat/model/extension/statemachines/excel/exporter/StateMachineExporter.java

```diff
@@ -34,57 +34,45 @@
 import de.dlr.sc.virsat.model.extension.statemachines.model.StateMachine;
 import de.dlr.sc.virsat.model.extension.statemachines.model.Transition;
 
-
-
-
 /**
  * Class for exporting excel
- * @author bell_er
- *
  */
 public class StateMachineExporter extends ExcelExportHelper implements IExport {
-	
+
 	CategoryAssignment exportCa;
-	
+
 	@Override
 	public void export(EObject eObject, String path, boolean useDefaultTemplate, String templatePath) {
-		if (eObject instanceof CategoryAssignment) {		
+		if (eObject instanceof CategoryAssignment) {
 			CategoryAssignment ca = (CategoryAssignment) eObject;
 			final String defaultTemplatePath = "/resources/StateMachineExportTemplate.xlsx";
 			// find the export template
 			try {
-				InputStream is = null;
+				InputStream iStream = null;
 				if (useDefaultTemplate) {
-					is = Activator.getResourceContentAsString(defaultTemplatePath);
+					iStream = Activator.getResourceContentAsString(defaultTemplatePath);
 				} else {
-					is = new FileInputStream(templatePath);
+					iStream = new FileInputStream(templatePath);
 				}
-				wb = new XSSFWorkbook(is);
-			} catch (IOException e) {
-				Status status = new Status(Status.ERROR, Activator.getPluginId(), "Failed to find the Template!", e);
-				Activator.getDefault().getLog().log(status);
-			}
-			
-			exportData(ca);
-			String newPath = path + "/" + ca.getFullQualifiedInstanceName() + ".xlsx";
-			// and write the results	
-			File file = new File(newPath);
-			try {
+				wb = new XSSFWorkbook(iStream);
+				exportData(ca);
+				String newPath = path + "/" + ca.getFullQualifiedInstanceName() + ".xlsx";
+				// and write the results
+				File file = new File(newPath);
+				// find the export destination
 				FileOutputStream out = new FileOutputStream(file);
 				wb.write(out);
 			} catch (IOException e) {
-				Status status = new Status(Status.ERROR, Activator.getPluginId(), "Failed to perform an export operation!", e);
+				Status status = new Status(Status.ERROR, Activator.getPluginId(), "Failed to perform an export operation! \n" + e.getMessage(), e);

```

Comment:
I think the error message will be printed anyways, so we can remove it from our message.

Reply:
There are two different Errors and I wanted to show the User where the problem is in the Dialogbox.